components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityKeyStoreResolverConstants.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.core.util;
+
+/**
+ * This class holds the constants used by IdentityKeyStoreManager.
+ */
+public class IdentityKeyStoreResolverConstants {
+    
+    // CustomKeyStoreMapping config path
+    public static final String CONFIG_ELEM_SECURITY = "Security";
+    public static final String CONFIG_ELEM_KEYSTORE_MAPPINGS = "CustomKeyStoreMappings";
+    public static final String CONFIG_ELEM_KEYSTORE_MAPPING = "CustomKeyStoreMapping";
+
+    // CustomKeyStoreMapping config attributes
+    public static final String ATTR_NAME_PROTOCOL = "Protocol";
+    public static final String ATTR_NAME_KEYSTORE_NAME = "KeyStoreName";
+    public static final String ATTR_NAME_USE_IN_ALL_TENANTS = "UseInAllTenants";
+
+    // KeyStore Properties
+    public static final String KS_PROPERTY_LOCATION = "Location";
+    public static final String KS_PROPERTY_TYPE = "Type";
+    public static final String KS_PROPERTY_PASSWORD = "Password";
+    public static final String KS_PROPERTY_KEY_ALIAS = "KeyAlias";
+    public static final String KS_PROPERTY_KEY_PASSWORD = "KeyPassword";
+
+    // KeyStore Constants
+    public static final String CUSTOM_KEY_STORE_SUFFIX = "KeyStore";
+    public static final String KEY_STORE_EXTENSION = ".jks";
+
+    // Inbound Protocols
```

Comment:
Double check with how this is used in other configs/docs

Reply:
Changed https://github.com/wso2/carbon-identity-framework/pull/5794/commits/439a2374299d5ce88f8dfa2b5a89c2683631228e